python/ray/util/state/state_cli.py

```diff
@@ -1103,10 +1103,8 @@ def log_actor(
         ```
 
     Raises:
-        :class:`RayStateApiException <ray.util.state.exception.RayStateApiException>`
```

Comment:
I think we need to add this back like in the other endpoints (it is probably thrown by one of the wrappers?)

Reply:
Whoops, fixing. Thanks!